docs/topics/test-page.md

```diff
@@ -17,7 +17,7 @@
 
 ```kotlin
 plugins {
-    kotlin("kapt") version "%kotlinVersion%"
```

Comment:
Why do we need to remove the variable? If it is a Docs functionality, I would like to keep it and adapt the tests.